core/trino-main/src/main/java/io/trino/operator/BigintGroupByHash.java

```diff
@@ -15,7 +15,6 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
-import io.trino.array.IntBigArray;
```

Comment:
Are there any microbenchmarks that you could attach as commit description?

Reply:
I attached the results. The change, however, makes more sense in synergy with next commits